ddsrecorder/src/cpp/main.cpp

```diff
@@ -26,8 +26,10 @@
 #include <cpp_utils/exception/ConfigurationException.hpp>
 #include <cpp_utils/exception/InitializationException.hpp>
 #include <cpp_utils/logging/CustomStdLogConsumer.hpp>
+#include <cpp_utils/logging/LogConfiguration.hpp>
 #include <cpp_utils/ReturnCode.hpp>
 #include <cpp_utils/time/time_utils.hpp>
+#include <cpp_utils/types/Fuzzy.hpp>
```

Comment:
It's not necessary to include Fuzzy.

Reply:
and for eprosima::utils::Fuzzy<eprosima::utils::VerbosityKind> log_verbosity?